go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -520,21 +542,25 @@ func (collector *TableGC) purge(ctx context.Context) (tableName string, err erro
 
 // dropTable runs an actual DROP TABLE statement, and marks the end of the line for the
 // tables' GC lifecycle.
-func (collector *TableGC) dropTable(ctx context.Context, tableName string) error {
-	conn, err := collector.pool.Get(ctx, nil)
+func (collector *TableGC) dropTable(ctx context.Context, tableName string, isBaseTable bool) error {
+	conn, err := dbconnpool.NewDBConnection(ctx, collector.env.Config().DB.DbaWithDB())
```

Comment:
Powerful user needed to issue a `DROP VIEW` because of the `DEFINER` of the view (the user dropping the view needs to be "stronger" than the dfiner).